PhotoPlusBackend/photoplus/src/main/resources/application.yml

```diff
@@ -33,8 +33,8 @@ spring:
 
   datasource:
     url: jdbc:mysql://localhost:3306/photoPlusDB?useSSL=false&serverTimezone=Europe/Warsaw
-    username: root
-    password: pass
+    username: photoPlus
```

Comment:
omege man why u do this

Reply:
"I don't want to but i have to." Fixed in ea48623